frontend/packages/console-shared/src/test-utils/utils.ts

```diff
@@ -149,13 +149,19 @@ export const waitForCount = (elementArrayFinder: any, targetCount: number) => {
 
 export const waitForStringInElement = (elem: any, needle: string) => {
   return async () => {
+    if (!(await elem.isPresent())) {
+      return false;
+    }
     const content = await elem.getText();
     return content.includes(needle);
   };
 };
 
 export const waitForStringNotInElement = (elem: any, needle: string) => {
```

Comment:
the `if (!(await elem.isPresent())) {` is good to check, because if the element doesn't happen to be present, the subsequent `await elem.getText();` would throw an exception.

As to whether we could use `!waitForStringInElement`, yes we could use it like that, I thought it would be a bit more readable this way. If you want to change the occurences to !waitForStringInElement instead, it can be done